jenkins/Jenkinsfile-release

```diff
@@ -18,11 +18,11 @@ node {
                 gitCheckout()
             }
 
-            stage('Build project') {
+            stage('Build BDK 2.0 project') {
                 sh './gradlew ' + releaseParamVersion
             }
 
-            stage('Deploy on OSSRH') {
+            stage('Deploy BDK2.0 on Nexus and Maven Central') {
                 sh './gradlew -PnexusUsername=symphony-platform-solutions -PnexusPassword=$PASSPHRASE ' +
                         '-Psigning.keyId=3C09C6D3 -Psigning.secretKeyRingFile=$KEYRING_FILE -Psigning.password=$PASSPHRASE ' +
```

Comment:
`-Psigning.keyId=3C09C6D3` is it safe to share it here? Is it the same value for everyone?

Reply:
Yes because it relates to the public key ID, https://docs.gradle.org/current/userguide/signing_plugin.html#sec:signatory_credentials says:
> The public key ID (The last 8 symbols of the keyId. You can use gpg -K to get it).